### PR TITLE
[webui] Allow edit kiwi package attributes

### PR DESCRIPTION
**Pull request by DavidKang**
Allow edit kiwi package attributes in the same way as in Kiwi repositories.
screenshot from 2017-08-02 17-00-57

Pair-programmed by: @Ana06 and @DavidKang.

### CODE REVIEW

**Comment by Ana06** [comment]
[(8) P3: Kiwi Editor Software Selection](https://trello.com/c/zsnpErV4/466-8-p3-kiwi-editor-software-selection)

**Comment by DavidKang** [comment]
@ChrisBr, should we remove some fields in the edit form?

File: src/api/app/assets/javascripts/webui/application/kiwi_editor.js [hunk 1]
```diff
@@ -21,17 +21,23 @@ function editDialog(){
...
   var fields = $(this).parents('.nested-fields');
-  var name = fields.find(".kiwi_repository_name");
+  var name = fields.find('.kiwi_element_name');
   var dialog = fields.find('.dialog');
-  var alias = dialog.find("[id$='alias']");
-  if (alias.val() != '') {
-    name.text(alias.val());
+  var name_package = dialog.find("[id$='name']").val();
+  if (name_package != '') {
+    name.text(name_package);
```
Comment (bgeuken) [inline src/api/app/assets/javascripts/webui/application/kiwi_editor.js]: @Ana06 @DavidKang nitpick: variables are usually written in camelcase in js

Comment (DavidKang) [inline src/api/app/assets/javascripts/webui/application/kiwi_editor.js]: @bgeuken, right! we will do it in another PR. Thans :smile_cat:
